### PR TITLE
chore(DEVOPS-2113): Version 7.15.2 added to the Dockerfile

### PR DESCRIPTION
**Pull request by ShaunGautham**
During the testing using the repo https://github.com/AlphaFounders/docker-elk.git , the memory usage was more than 95% with the version 8.15.0. When the version is switched to 7.15.2, the memory usage is dropped to less than 60%

### CODE REVIEW

**Review by nitheesb — APPROVED** [review]

File: elasticsearch/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG ELK_VERSION
 
 # https://www.docker.elastic.co/
-FROM docker.elastic.co/elasticsearch/elasticsearch:${ELK_VERSION}
+FROM docker.elastic.co/elasticsearch/elasticsearch:7.15.2
```
Comment (mithRabbit) [inline elasticsearch/Dockerfile]: can you update the ENV instead not here

Comment (gowriter) [reply elasticsearch/Dockerfile:4]: i would recommend getting rid of this useless env var, dev should decide what version. approved!

**Review by gowriter — APPROVED** [review]

File: elasticsearch/Dockerfile (hunk 1 above)
Comment (gowriter) [reply elasticsearch/Dockerfile:4]: After discussing this change with mithlash, i agree with him, should update the env

**Review by JeremyRms — CHANGES_REQUESTED** [review]
Agreed with Mithlesh, please update the env file instead

**Review by JeremyRms — APPROVED** [review]

**Pull request merged**